packages/core-utils/src/types.js

```diff
@@ -120,6 +135,11 @@ export const configType = PropTypes.shape({
          * vehicle rental overlays.
          */
         modes: PropTypes.arrayOf(PropTypes.string.isRequired),
+        /**
+         * The applicable map symbols. Only applicable in vehicle rental
+         * overlays.
+         */
+        symbols: zoomBasedSymbolType,
```

Comment:
I changed the prop name because the underlying type is different, so that would force people to make the proper change. On the other hand, if we need to use definitions from yaml files, we could support both mapSymbols and symbols props for the vehicle rentals (as it was at at commit https://github.com/opentripplanner/otp-ui/commit/ad7327bfa5e071bcfc79b8c50f4f23cf764e6910), and that way there would be no compatibility issues. Thoughts @landonreed @evansiroky ?

Reply:
Addressed with commits https://github.com/opentripplanner/otp-ui/pull/159/commits/5bda99ea5ece869a3385ea4bc84a081591f7b379 and https://github.com/opentripplanner/otp-ui/pull/159/commits/e16b6172f75edd285260e6a256df63cf0e2d1e34.